### PR TITLE
Fix #323 Error communicating with OpenAI: Invalid URL 'your-base-url-for-azure/embeddings' 

### LINKED ISSUE #323: Error communicating with OpenAI: Invalid URL 'your-base-url-for-azure/embeddings'
### Duplicates

- [x] I have searched the existing issues

### Steps to reproduce 🕹

I get this error after describing the goals of my AI. It does not seem to matter what is entered.

### Current behavior 😯

After entering my goals I get this error: Error communicating with OpenAI: Invalid URL 'your-base-url-for-azure/embeddings'

I have USE_AZURE=False in .env. I tried removing the AZURE api lines, with the same result.

### Expected behavior 🤔

It should not cause the error.

### Your prompt 📝

```yaml
❯ python3 scripts/main.py --continuous --gpt3only --speak                                                       ─╯
Continuous Mode:  ENABLED
WARNING:  Continuous mode is not recommended. It is potentially dangerous and may cause your AI to run forever or carry out actions you would not usually authorise. Use at your own risk.
Speak Mode:  ENABLED
GPT3.5 Only Mode:  ENABLED
Welcome back!  Would you like me to return to being doctor-zimmerman?
Continue with the last settings?
Name:  doctor-zimmerman
Role:  an ai designed to look up what time it is.
Goals: ['get the current time', 'shut down']
Continue (y/n): y
Using memory of type: PineconeMemory
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_requestor.py", line 516, in request_raw
    result = _thread_context.session.request(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/requests/sessions.py", line 573, in request
    prep = self.prepare_request(req)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/requests/sessions.py", line 484, in prepare_request
    p.prepare(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/requests/models.py", line 368, in prepare
    self.prepare_url(url, params)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/requests/models.py", line 439, in prepare_url
    raise MissingSchema(
requests.exceptions.MissingSchema: Invalid URL 'your-base-url-for-azure/embeddings': No scheme supplied. Perhaps you meant https://your-base-url-for-azure/embeddings?

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/gavinblair/Projects/personal/autogpt/scripts/main.py", line 295, in <module>
    assistant_reply = chat.chat_with_ai(
                      ^^^^^^^^^^^^^^^^^^
  File "/Users/gavinblair/Projects/personal/autogpt/scripts/chat.py", line 69, in chat_with_ai
    relevant_memory = permanent_memory.get_relevant(str(full_message_history[-5:]), 10)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gavinblair/Projects/personal/autogpt/scripts/memory.py", line 55, in get_relevant
    query_embedding = get_ada_embedding(data)
                      ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gavinblair/Projects/personal/autogpt/scripts/memory.py", line 10, in get_ada_embedding
    return openai.Embedding.create(input=[text], model="text-embedding-ada-002")["data"][0]["embedding"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_resources/embedding.py", line 33, in create
    response = super().create(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_resources/abstract/engine_api_resource.py", line 153, in create
    response, _, api_key = requestor.request(
                           ^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_requestor.py", line 216, in request
    result = self.request_raw(
             ^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_requestor.py", line 528, in request_raw
    raise error.APIConnectionError(
openai.error.APIConnectionError: Error communicating with OpenAI: Invalid URL 'your-base-url-for-azure/embeddings': No scheme supplied. Perhaps you meant https://your-base-url-for-azure/embeddings?
```

### PR DESCRIPTION
**Pull request by pratiksinghchauhan**
### Background
As stated in issue #323, users are getting the following error

`Error communicating with OpenAI: Invalid URL 'your-base-url-for-azure/embeddings'
`
The cause of this issue is the clashing of the variable name `OPENAI_API_BASE` between AUTO_GPT's [.env.template](https://github.com/Torantulino/Auto-GPT/blob/4b7f116455ff42cf34487f22f03be308c4851382/.env.template#L10) and OpenAI [official python library](https://github.com/openai/openai-python/blob/2b0bb90712f48246a7dce5a755a7e93b169da6f0/openai/__init__.py#L37). So once the environment loads it takes OPENAI_API_BASE from .env of AUTO-GPT which it finds to be `your-base-url-for-azure ` and starts giving out this error. Please click on the hyperlinks to know more.

To fix this I have renamed the variable `OPENAI_API_BASE` to `OPENAI_AZURE_API_BASE`. More details are in the changes section.

### Changes
For fixing the issue and for better readability and understanding I have done the following changes,
- Renamed env variable `OPENAI_API_BASE` to `OPENAI_AZURE_API_BASE`.
- Renamed env variable `OPENAI_API_VERSION` to `OPENAI_AZURE_API_VERSION`
- Renamed env variable `OPENAI_DEPLOYMENT_ID` to `OPENAI_AZURE_DEPLOYMENT_ID`

The latter two variables did not pose any issue earlier but for better readability and consistency, I have renamed them too.

### Documentation
I have done changes in README.md file for documenting my changes

### Test Plan
I have tested this using multiple prompts, and also by making `USE_AZURE` to True, and in both the cases the PR works fine.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thouroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: scripts/commands.py [hunk 1]
```diff
@@ -165,8 +165,9 @@ def google_official_search(query, num_results=8):
     return search_results_links
 
 def browse_website(url, question):
-    summary = get_text_summary(url, question)
-    links = get_hyperlinks(url)
+    website_content = browse.get_website_content(url)
```
Comment (richbeales) [inline scripts/commands.py]: please remove these changes for commands.py to keep the PR atomic and related to the title of the PR

**Review by richbeales — CHANGES_REQUESTED** [review]
please drop the changes to commands.py from this PR as they're not relevant.

File: scripts/commands.py (hunk 1 above)
Comment (pratiksinghchauhan) [reply scripts/commands.py]: @richbeales my apologies, I have dropped the changes. It came from my previous PR.  I think it should be fine now.

**Review by nponeccop — APPROVED** [review]

**Review by anton264 — APPROVED** [review]

**Review by karthikR10 — APPROVED** [review]

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
@pratiksinghchauhan Conflicts again. We are merging minor PRs

**Comment by pratiksinghchauhan** [comment]
@nponeccop I have fixed the conflicts.

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
I guess rename the title to something like "Clarify Azure config variables", the original title is no longer relevant.

**Comment by pratiksinghchauhan** [comment]
@nponeccop Thanks looking forward to working on this.

**Review by Torantulino — APPROVED** [review]

**Pull request merged**